external/fv3fit/fv3fit/train_microphysics.py

```diff
@@ -204,6 +208,93 @@ def __post_init__(self) -> None:
             self.cache = False
 
 
+def get_model_output_sensitivities(
+    model: tf.keras.Model, sample: Mapping[str, tf.Tensor],
+):
+    """
+    Generate sensitivity jacabion for each output relative to a mean
+    value for each input
```

Comment:
Please describe outputs and the normalization strategy (e.g. what the units of the outputs)?